CRM/Extension/Browser.php

```diff
@@ -157,21 +157,31 @@ public function getExtensions() {
 
   /**
    * Get a description of a particular extension.
+   * This function is an optimised form of getExtensions() - it returns the
```

Comment:
While I like being descriptive I think this comment doesn't add too much, it's more of a description of what you're changing here. In the future if someone looks at this it should be clear that this is separate from `getExtensions`